ZuluContent/Zulu/Items/SingleClick/SingleClickHandler.cs

```diff
@@ -59,7 +66,7 @@ private static bool Validate(Mobile m, Item item)
             if (!ClilocList.Entries.TryGetValue(item.LabelNumber, out var desc))
                 return false;
 
-            if (item is IMagicItem magicItem && !magicItem.Identified)
+            if (item is IMagicItem magicItem && m.AccessLevel == AccessLevel.Player && !magicItem.Identified)
```

Comment:
Might take this out eventually (or use a DEBUG flag?), but it helped a lot with debugging the loot ingame

Reply:
A flag to toggle this would be useful. I initially had something similar for not requiring GMs to id items but I needed to test identification worked correctly as well. For now we can probably just flip it off and on manually.

Reply:
cool I'll add a flag then